tmexp/__main__.py

```diff
@@ -47,9 +47,30 @@ def add_force_arg(cmd_parser: argparse.ArgumentParser) -> None:
 def add_dataset_arg(cmd_parser: argparse.ArgumentParser) -> None:
     cmd_parser.add_argument(
         "--dataset-name",
-        help="Name of the dataset, used as the second level input/output directory.",
+        help="Name of the dataset created by `preprocess`, defaults"
+        " to 'MM-DD-HH:MM-dataset'.",
+        default=None,
```

Comment:
```suggestion
        default=check_create_default("dataset"),
```
with the mentioned edits to this function later in the review.

Reply:
done here and elsewhere